PlexRequests.UI/Views/Admin/Sonarr.cshtml

```diff
@@ -222,13 +264,52 @@
                 dataType: "json",
                 success: function (response) {
                     response.forEach(function (result) {
-        				$('#getSpinner').attr("class", "fa fa-check");
+                        $('#getSpinner').attr("class", "fa fa-check");
                         $("#select").append("<option value='" + result.id + "'>" + result.name + "</option>");
                     });
                 },
                 error: function (e) {
                     console.log(e);
-        				$('#getSpinner').attr("class", "fa fa-times");
+                    $('#getSpinner').attr("class", "fa fa-times");
+                    generateNotify("Something went wrong!", "danger");
+                }
+            });
+        });
+
+        $('#getRootFolders').click(function (e) {
+
+            $('#getRootFolderSpinner').attr("class", "fa fa-spinner fa-spin");
```

Comment:
See my comment in the Module, we can move the validation using Nancy's Validate stuff (see here https://github.com/NancyFx/Nancy/wiki/Nancy-and-Validation)